packages/client/vite.config.ts

```diff
@@ -50,45 +51,20 @@ const getProjectConfigExtensions = async (config) => {
     if (fs.existsSync(staticPath)) {
       const { default: viteConfigExtension } = require(staticPath)
       if (typeof viteConfigExtension === 'function') {
+        console.log(viteConfigExtension)
         const configExtension = await viteConfigExtension()
-        config.plugins = [...config.plugins, ...configExtension.default.plugins]
+        config.plugins = [...config.plugins!, ...configExtension.default.plugins]
```

Comment:
The array closing bracket after `...config.plugins` is by mistake? Because I see 2 closing brackets while 1 opening brackets.

Reply:
that is an exclamation point to stop typescript complaining ;)